Add cloud info to user guidance (#122)
WCRP-CMIP/cmip7-guidance: docs/CMIP7/Guidance_for_users.md

Reviewer: ```suggestion
[emd]:  https://wcrp-cmip.github.io/Essential-Model-Documentation/docs/
```

Reviewer: ```suggestion
[varlist]: https://cmip-data-request.github.io/cmip7-dreq-webview/latest/variables.html
```

Reviewer: ```suggestion
Models report data on a variety of time steps. In CMIP6, MIP tables (aka CMOR tables) defined the frequency at which requested variables in an experiment should be reported (e.g., "Amon", "day"). In CMIP7 the frequency is specified in the Data Request for every requested variable and reported in netCDF files as a global attribute. The [CMOR tables for CMIP7](https://github.com/WCRP-CMIP/cmip7-cmor-tables) contain branded variables, are organized by realm, and no longer specify frequency.
```

Reviewer: ```suggestion
[freqlist]: https://cmip-data-request.github.io/cmip7-dreq-webview/latest/cmip7_frequency.html
```

Reviewer: ```suggestion
[grid]: https://doi.org/10.5281/zenodo.15697024
```
(link that always points at latest version)

Reviewer: ```suggestion
[GlobalAttrs]: https://doi.org/10.5281/zenodo.17250296
```
(link that always points at latest version)

# CMIP7 Guidance for Data Users

> This page is designed to inform users of climate model outputs on key CMIP7 concepts and tools. It is a landing page to provide context and to redirect them to the proper resources to learn more.

Reviewer: ```suggestion
This page is designed to inform users of climate model outputs on key CMIP7 concepts and tools. It is a landing page to provide context and to redirect them to more detailed resources to learn more.
```

??? info "Understanding ESGF Nodes"

>     ESGF is a collaboration of groups, agencies and institutions around the world, that are dedicated to the development and operation of a long-term system for the management, access and analysis of climate data. The ESGF architecture is based on a system of autonomous and distributed Nodes. Data is hosted on a collection of data nodes located at modelling centres or data centres across the world. Nodes exchange information about their data holdings and services, trust each other for registering users and establishing access control decisions. Data can be searched through index nodes that hold catalogues of what is available on the data node. The net result is that a user can use a web browser or rich desktop client, connect to any Node, and seamlessly find and access data throughout the federation. 

Reviewer 2: ```suggestion
    ESGF is a collaboration of groups, agencies and institutions around the world, that are dedicated to the development and operation of a long-term system for the management, access and analysis of climate data.  The ESGF architecture is based on a system of distributed Nodes.  Data is hosted on a collection of data nodes located at modelling centres or data centres across the world.  Data can be searched through index nodes that hold catalogues of what is available on the data node. The net result is that a user can use a web browser or rich desktop client, connect to any Node, and seamlessly find and access data throughout the federation. 
```

Reviewer 2: The sentence I removed no longer applies to the current ESGF node architecture (long story regarding that functionality)

    While all published CMIP7 data is available from ESGF, some of it is additionally hosted in non-ESGF storage facilities. Below are links to some of these replicas. If you know of another place CMIP data is currently being stored, please submit [this form][altaccess] to let us and the community know!

>     * The [Pangeo / ESGF Cloud Data Working Group][pangeo] is working on providing cloud access for CMIP7 ARCO data. 

Reviewer 3: ```suggestion
```

>     For all non-ESGF data access routes, we encourage users to verify that the data used is the latest version. 

Reviewer 3: ```suggestion
```

Author: would a future CMIP7 subset in the cloud  be guaranteed to have the latest version ?

Author: ah I see the doubling nevermind

    ??? info "Analysis-Ready Cloud-Optimized data"

>         Data hosted in the cloud can be very useful for users with fewer ressources. Indeed, data can be streamed and the analysis can be done remotely close to the storage, instead of downloading it locally. For efficient analysis in the cloud, the data needs to be in an Analysis-Ready Cloud-Optimized (ARCO) format. The Pangeo / ESGF Cloud Data Working Group are working on tools (e.g., [virtualizarr][virtualizarr]) for this purpose. Contributions are welcomed [here][virtual].

Reviewer 3: ```suggestion
        Data hosted in the cloud can be very useful for users with fewer resources. Indeed, data can be streamed and the analysis can be done remotely close to the storage, instead of downloading it locally. For efficient analysis in the cloud, the data needs to be in an Analysis-Ready Cloud-Optimized (ARCO) format. Following the Pangeo / ESGF Cloud Data Working Group efforts for CMIP6,  there are community efforts underway to streamline ARCO access (e.g., [virtualizarr][[virtualizarr](https://virtualizarr.readthedocs.io/en/stable/)]) to lowering barriers to producing cloud-optimized data formats without having to rewrite the original data. Contributions are welcomed [here][virtual].
      
For data access routes that do not involve official ESGF nodes, we encourage users to verify that the data used is the latest version, and cite the original sources appropriately. 

```

* grid
* version

!!! Info

    Current advice from the CVs task team is to only access the CVs via [ESGVOC](https://esgf.github.io/esgf-vocab/). This will be subject to change in the future.

More information about the meaning of these facets is provided in the [global attributes documentation][GlobalAttrs], with further guidance provided on the [Global Attributes page](Global_Attributes.md). The values associated with each facet are standardized through the [CVs][cmipCvs]. They are used to search the ESGF database and can be found in the global attributes of the data. This section provides helpful links and gives a bit more information on a few key facets. 

### 3.1.  Source and Variant
* [List of models][sourcelist] (coming soon)
* [Essential Model Documentation (EMD)][emd] (coming soon)

> The Essential Model Documentation (EMD) contains a high-level description intended to contain information on model formulation that can be easily compared between different models. EMD pages contain links to more in-depth model documentation for each source.

Reviewer: ```suggestion
The Essential Model Documentation (EMD) contains a high-level description intended to contain information on model formulation that can be easily compared between different models. The [EMD guidance pages][emd] contain links to more in-depth model documentation for each source.
```

### 3.3. Variable
* [List of variables][varlist]
* [Branded variable documentation](Branded_Variables.md)

In CMIP7, the concept of branded variables has been introduced to make it easier to find the variables you want. Branded variables follow the template: 

```
<branded_variable> 
= <variable_id>_<branding_suffix>
= <variable_id>_<temporal_label>-<vertical_label>-<horizontal_label>-<area_label>
```

To uniquely identify requested variables, frequency and region are required together with the branded variable.
The CMIP7 compound name used in the Data Request also includes each requested variable's primary realm, for ease of identifying variables associated with a realm:
```
<compound_name> = <realm>.<branded_variable>.<frequency>.<region>
```

For example, the equivalent of `Amon.tas` in CMIP6 would be  `atmos.tas_tavg-h2m-hxy-u.mon.GLB` in CMIP7.

> ??? info "Understanding Data Request"

Reviewer: ```suggestion
??? info "Understanding the Data Request"
```

> * [List of variables][varlist] (coming soon)

Reviewer: ```suggestion
* [List of variables][varlist]
```
(link added below)

* [Branded variable documentation](Branded_Variables.md)

> The variables produced in CMIP7 were recommended by the [CMIP7 Data Request task team][DataRequestTeam]. In CMIP7, the concept of branded variable identifies the variables. Branded variables follows the template: 

Reviewer: ```suggestion
The variables produced in CMIP7 were recommended by the [CMIP7 Data Request task team][DataRequestTeam]. The latest version of the Data Request can be [viewed here](https://bit.ly/CMIP7-DReq-latest), and further guidance can be [found here](https://wcrp-cmip.github.io/cmip7-guidance/docs/CMIP7/Guidance_for_modellers/#4-model-output-fields). In CMIP7, the concept of branded variables has been introduced to make it easier to find the variables you want. Branded variables follow the template: 
```

```
<branded_variable> 

> = <variable_id>_<branded_suffix>

Reviewer: ```suggestion
= <variable_id>_<branding_suffix>
```
(my bad: in the earlier version of the branded variables guidance page I used "branded suffix", but it should be "branding suffix")

= <variable_id>_<temporal_label>-<vertical_label>-<horizontal_label>-<area_label>
```

> To uniquely identify variables, frequency and region are required together with the branded variable.

Reviewer: ```suggestion
To uniquely identify requested variables, frequency and region are required together with the branded variable.
```

> The CMIP7 compound name also includes realm for ease of use:

Reviewer: ```suggestion
The CMIP7 compound name used in the Data Request also includes each requested variable's primary realm, for ease of identifying variables associated with a realm:
```

```
<compound_name> = <realm>.<branded_variable>.<frequency>.<region>
```

For example, the equivalent of `Amon.tas` in CMIP6 would be  `atmos.tas_tavg-h2m-hxy-u.mon.GLB` in CMIP7.

<!--TODO: maybe add more about Data Request. -->

### 3.4 Frequency

> * [List of frequencies][freqlist] (coming soon)

Reviewer: ```suggestion
* [List of frequencies][freqlist]
```
(link added below)

## 6. New to CMIP?

> First time using CMIP? Need a bit more help ? Check out these ressources:

Reviewer 2: ```suggestion
First time using CMIP? Need a bit more help ? Check out these resources:
```

Reviewer 2: typo

* [Entry-Level Documentation][eld] (coming soon), put together by the [Fresh Eyes on CMIP][FeoC] group.
* [List of tools for using climate data][tools].
* You have a more specific question ? Ask it on the [Fresh Eyes Platform][platform]. (You need to register [here][register] first.)

> ###### Document version: 2025-10-08

Reviewer: ```suggestion
###### Document version: 2026-04-08
```
